dashboard/app/controllers/courses_controller.rb

```diff
@@ -155,6 +155,14 @@ def code
 
   def standards
     unit_group = UnitGroup.get_from_cache(params[:course_name])
+    if !unit_group.present? && UnitGroup.family_names.include?(params[:course_name])
+      redirect_to_course = UnitGroup.all_courses.
+        select {|c| c.family_name == params[:course_name] && c.stable?}.
+        sort_by(&:version_year).
+        last
```

Comment:
any reason not to pull this and https://github.com/code-dot-org/code-dot-org/blob/d86cd51e40578bf02853cd377503e7c3507f49bb/dashboard/app/controllers/courses_controller.rb#L73-L76 out into a helper method? Something like `redirect_to_course = UnitGroup.latest_stable(params[:course_name])`